qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/1-mount

```diff
@@ -0,0 +1 @@
+.qa/cephfs/mount/
```

Comment:
But I'm generally happy to see our suites consistently use a link to common yamls.

Reply:
This moves the mount task to before the file system is created by `1-volume`. I don't think that's intentional?